src/Nether.Analytics.DataLake/Scheduler/JobExecutor.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Generic comment for anything that doesn't necessary have anything to do with DataLake. Move to Nether.Analytics